### PR TITLE
Add 1-click-bom file

### PR DESCRIPTION
**Pull request by kasbah**
This can be used with the [1-click BOM browser extension](https://1clickBOM.com) to quickly put all the parts into the Digikey cart. 

Also, if someone turns it into a PCB it would be really nice to add it as a project on [kitnic.it](https://kitnic.it).

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -93,6 +93,8 @@ Full schematics:
 
 Part list:
 
+You can easily buy the parts listed in the [1-click-bom](1-click-bom.tsv) file using the [1-click BOM extension](https://1clickbom.com).
```
Comment (ashtuchkin) [inline readme.md]: Could you make this take less vertical space by moving the link to the part list line? I.e.
```
Part list (add to cart with [1-click BOM](https://1clickbom.com)):
```

**Review by ashtuchkin — CHANGES_REQUESTED** [review]
Hey thanks for the pull request, I think it will be useful for others! Please see minor comment inline.

**Comment by kasbah** [comment]
Thank _you_! I changed the line but also made it link to the tsv file.

**Comment by ashtuchkin** [comment]
Thank you!

**Pull request merged**